# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1305696665 25200
#      Tue May 17 22:31:05 2011 -0700
# Branch stable
# Node ID 486c9aea34d322a980b91df40fbac293254d4b92
# Parent  c21eb9c02c3139cbc7e201bbd0c505ea4edbb7d0
doc: Correctly refer to toupper() as a Mapping Function.

* mappers.cc (toupper): Change "Built-in Function" to "Mapping Function"

diff --git a/src/mappers.cc b/src/mappers.cc
--- a/src/mappers.cc
+++ b/src/mappers.cc
@@ -1828,18 +1828,18 @@ DEFALIAS (lower, tolower);
 %!  a(3,3,3,3) = "D";
 %!  assert(tolower(a)(3,3,3,3), "d");
 
 */
 
 
 DEFUNX ("toupper", Ftoupper, args, ,
     "-*- texinfo -*-\n\
-@deftypefn  {Built-in Function} {} toupper (@var{s})\n\
-@deftypefnx {Built-in Function} {} upper (@var{s})\n\
+@deftypefn  {Mapping Function} {} toupper (@var{s})\n\
+@deftypefnx {Mapping Function} {} upper (@var{s})\n\
 Return a copy of the string or cell string @var{s}, with each lowercase\n\
 character replaced by the corresponding uppercase one; non-alphabetic\n\
 characters are left unchanged.  For example:\n\
 \n\
 @example\n\
 @group\n\
 toupper (\"MiXeD cAsE 123\")\n\
      @result{} \"MIXED CASE 123\"\n\
